### PR TITLE
feat: Added Toast Component

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
Types of Toast.
- Baseline.
- Leading
- Stacked

### CODE REVIEW

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: Components/Toasts/toast.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    opacity: 0.5;
+}
+
+
+/*-----------------------------------------*/
+
+.stackedToastContainer {
+    border-radius: 0.5rem;
+    padding: 0.5rem 1rem;
+    background-color: black;
```
Comment (abhij1607) [inline Components/Toasts/toast.css]: color variables could be used

**Review by abhij1607 — APPROVED** [review]
looks good

File: Components/Toasts/toast.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply Components/Toasts/toast.css]: Thanks for the suggestion

**Pull request merged**